ZConfig/tests/test_schema2html.py

```diff
@@ -0,0 +1,87 @@
+##############################################################################
+#
+# Copyright (c) 2017 Zope Foundation and Contributors.
+# All Rights Reserved.
+#
+# This software is subject to the provisions of the Zope Public License,
+# Version 2.1 (ZPL).  A copy of the ZPL should accompany this distribution.
+# THIS SOFTWARE IS PROVIDED "AS IS" AND ANY AND ALL EXPRESS OR IMPLIED
+# WARRANTIES ARE DISCLAIMED, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+# WARRANTIES OF TITLE, MERCHANTABILITY, AGAINST INFRINGEMENT, AND FITNESS
+# FOR A PARTICULAR PURPOSE.
+#
+##############################################################################
+from __future__ import absolute_import
+
+import contextlib
+import sys
+import unittest
+
+try:
+    # Note that we're purposely using the old
+    # StringIO object on Python 2 because it auto-converts
+    # Unicode to str, which io.BytesIO and io.StringIO don't
+    # but which normal printing to default sys.stdout *does*
+    from cStringIO import StringIO
+except ImportError:
+    from ZConfig._compat import NStringIO as StringIO
+
+
+from ZConfig import schema2html
+
+from .support import input_file
+from .support import with_stdin_from_input_file
+
+
+@contextlib.contextmanager
+def stdout_replaced(buf):
```

Comment:
Imagine hiding something like that in the documentation.  I guess those of us who've been around long enough should know to check for that rather than relying on memory.